staff/templates/instafeed.html

```diff
@@ -0,0 +1,25 @@
+
+<div id="mui-panel" class="rightpanelbody-hidden">
+      <h2 style="text-align: center; padding-bottom: 10px; font-size: 35px; color: #474747;">Instagram Feed</h2><hr style="width: 90%; height:2px;border:none;color:gray;background-color:gray; opacity: 0.2" />
+      <div id="instafeed">
+        <!-- INSTAFEED STUFF DOWN HERE !!!DO NOT CODE DOWN BELOW!!! -->
+      </div>
+</div>
+
+<div class="img-display-container" onclick="this.style.display='none'" style="position: fixed; width: 100%; height: 100%; top: 62px; background: rgba(0, 0, 0, 0.6); z-index: 1; text-align: center; vertical-align: middle">
+  <span onclick="this.parentElement.style.display='none'" class="closebtn">&times;</span>
```

Comment:
This onclick should be redundant because the parent does the same thing?

Reply:
Resolved in 66b87b02879fa1455def1a4ac386257eb03e01a6